node/src/ML.cpp

```diff
@@ -24,7 +26,18 @@ namespace node {
     }
 
     Napi::Value ML::CreateContext(const Napi::CallbackInfo& info) {
-        Napi::Object context = Context::constructor.New({});
+        // std::cout<<info.Length()<<std::endl;
+        WEBNN_NODE_ASSERT(info.Length() <= 1, "The number of arguments is invalid.");
+
+        Napi::Object context;
```

Comment:
```
std::vector<napi_value> args = {};
if (info.Length() > 0) args.push_back(info[0].As<Napi::Object>());
return Context::constructor.New(args);
```

Reply:
I do not think this implementation check if the parameter is a Object. So I use a similar way in the next commit. Please check it.